File: Eventos.php
<?php
header('Content-Type: application/json');
$pdo=new PDO("mysql:dbname=sistema;localhost","root","");

$accion= (isset($_GET['accion']))?$_GET['accion']:'leer';

switch($accion){
  case 'agregar':
    // Agregadp
       $sentenciaSQL = $pdo->prepare("INSERT INTO eventos (title,descripcion,color,textColor,start,endd)
       VALUES(:title,:descripcion,:color,:textColor,:start,:endd)");

       $respuesta=$sentenciaSQL ->execute(array(
         "title" => $_POST['title'],
         "descripcion" => $_POST['descripcion'],
         "color" => $_POST['color'],
         "textColor" => $_POST['textColor'],
         "start" => $_POST['start'],
         "endd" => $_POST['endd']
       ));
  echo json_encode($respuesta);
    break;
  case 'eliminar':
  //Eliminar
  // echo "Eliminar";
$respuesta=false;
if(isset($_POST['id'])){

  $sentenciaSQL= $pdo->prepare("DELETE FROM eventos WHERE ID=:ID");
  $respuesta= $sentenciaSQL->execute(array("ID"=>$_POST['id']));
}
echo json_encode($respuesta);
    break;

  case 'modificar':
 //Modificar
    $sentenciaSQL = $pdo->prepare("UPDATE eventos SET
    title=:title,
    descripcion=:descripcion,
    color=:color,
    textColor=:textColor,
    start=:start,
    endd=:endd
    WHERE ID=:ID
    ");

    $respuesta=$sentenciaSQL ->execute(array(
      "ID" =>$_POST['id'],
      "title" => $_POST['title'],
      "descripcion" => $_POST['descripcion'],
      "color" => $_POST['color'],
      "textColor" => $_POST['textColor'],
      "start" => $_POST['start'],
      "endd" => $_POST['endd']
    ));
    echo json_encode($respuesta);
    break;
  default:
        $sentenciaSQL= $pdo->prepare("SELECT * FROM eventos");
        $sentenciaSQL->execute();

        $resultado=$sentenciaSQL->fetchAll(PDO::FETCH_ASSOC);
        echo json_encode($resultado);  // code...
    break;
}


  ?>

File: eventos.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Text;
namespace Sistema
{
    #region Eventos
    public class Eventos
    {
        #region Member Variables
        protected int _id;
        protected string _title;
        protected string _descripcion;
        protected string _color;
        protected string _textColor;
        protected DateTime _start;
        protected DateTime _endd;
        #endregion
        #region Constructors
        public Eventos() { }
        public Eventos(string title, string descripcion, string color, string textColor, DateTime start, DateTime endd)
        {
            this._title=title;
            this._descripcion=descripcion;
            this._color=color;
            this._textColor=textColor;
            this._start=start;
            this._endd=endd;
        }
        #endregion
        #region Public Properties
        public virtual int Id
        {
            get {return _id;}
            set {_id=value;}
        }
        public virtual string Title
        {
            get {return _title;}
            set {_title=value;}
        }
        public virtual string Descripcion
        {
            get {return _descripcion;}
            set {_descripcion=value;}
        }
        public virtual string Color
        {
            get {return _color;}
            set {_color=value;}
        }
        public virtual string TextColor
        {
            get {return _textColor;}
            set {_textColor=value;}
        }
        public virtual DateTime Start
        {
            get {return _start;}
            set {_start=value;}
        }
        public virtual DateTime Endd
        {
            get {return _endd;}
            set {_endd=value;}
        }
        #endregion
    }
    #endregion
}